test/gui/tst_moveFilesFolders/test.feature

```diff
@@ -41,4 +41,17 @@ Feature: move file and folder
         And as "Alice" folder "folder1/folder2/folder3/folder4/folder5/test-folder2" should exist on the server
         And as "Alice" file "testFile.txt" should not exist on the server
         And as "Alice" folder "test-folder1" should not exist on the server
-        And as "Alice" folder "test-folder2" should not exist on the server
\ No newline at end of file
+        And as "Alice" folder "test-folder2" should not exist on the server
+
+
+    Scenario: Rename a file and a folder
+        Given user "Alice" has uploaded file with content "test file 1" to "textfile.txt" on the server
+        And user "Alice" has set up a client with default settings
+        When the user renames a file "textfile.txt" to "lorem.txt"
+        And the user renames a folder "folder1" to "FOLDER"
+        And the user waits for file "lorem.txt" to be synced
+        And the user waits for folder "FOLDER" to be synced
```

Comment:
Do we have a step to wait for syncing all files? I think that was complicated/ impossible to implement so we avoided that. @saw-jan  right?